src/lib/features/dependent-features/dependent-features-read-model.ts

```diff
@@ -42,6 +43,21 @@ export class DependentFeaturesReadModel implements IDependentFeaturesReadModel {
             variants: row.variants,
         }));
     }
+    async getFeatureDependenciesByChildren(
+        children: string[],
+    ): Promise<FeatureDependency[]> {
+        const rows = await this.db('dependent_features').whereIn(
+            'child',
+            children,
+        );
+
+        return rows.map((row) => ({
+            child: row.child,
```

Comment:
you can add this to the IDependency and use getParents method instead without growing API surface area